docs/reference/commandline/pull.md

```diff
@@ -45,7 +45,7 @@ use `docker pull`:
     # and the un-tarred base).
     $ docker pull --all-tags centos
     # will pull all the images from the centos repository
-    $ docker pull registry.hub.docker.com/debian
+    $ docker pull docker.io/library/debian
```

Comment:
Can we make one up for example purposes?

I agree with @dmp42 that it's better to avoid `docker.io` in the docs. Also, since this example involves an official image, it has the weird `/library` special case, which could be confusing.